R/utils-limits.R

```diff
@@ -47,14 +46,12 @@ testCallouts <- function(plot_obj, xlimits){
 #' 
 testCalloutsByView <- function(plot_obj, view_num, xlimits_real, width_char, xrange){
   
-  i_view <- which(names(plot_obj$view_num) == "callouts")
+  i_view <- which(names(plot_obj[[view_num]]) == "callouts")
```

Comment:
I think `view_num` needs to be wrapped in quotes here when not using the `$` operator, otherwise it treats it as a variable. I think the `$` adds implicit quotes. I'm not totally sure about that though. @lindsaycarr?